guacamole/src/main/frontend/src/translations/en.json

```diff
@@ -47,6 +47,8 @@
         "FIELD_HEADER_PASSWORD_AGAIN" : "Re-enter Password:",
         "FIELD_HEADER_RECORDING_WRITE_EXISTING" : "Allow writing to existing recording file:",
 
+        "FIELD_HEADER_TYPESCRIPT_WRITE_EXISTING" : "Allow writing to existing typescript file:",
```

Comment:
#Nitpick - can you remove the blank line above this one?

Reply:
Sure; it's done.